config/builder.yaml

```diff
@@ -6,6 +6,22 @@ spec:
   group: kpack.io
   versions:
   - name: v1alpha1
+    served: true
+    storage: false
```

Comment:
Should we store this version?

Reply:
We can only store one version and I think it is easier to store `v1alpha1` objects as `v1alpha2` objects (e.g. no new features are used) than to store `v1alpha2`  as `v1alpha1` (any new feature must be stored in the annotation to not get lost).

Reply:
Yep, that makes sense. I read that wrong.